challenge.md

```diff
@@ -5,19 +5,23 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git clone "ssh url"
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
-
+git status
```

Comment:
`git status` will let us know which files have been changed since the last commit. It will not let us know what the actual changes are. In order to see the changes line-by-line since the last commit, we'd use `git diff HEAD`. Partial credit.